src/main/java/com/github/instagram4j/instagram4j/requests/accounts/AccountsEditStatusRequest.java

```diff
@@ -0,0 +1,55 @@
+package com.github.instagram4j.instagram4j.requests.accounts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.instagram4j.instagram4j.IGClient;
+import com.github.instagram4j.instagram4j.models.IGPayload;
+import com.github.instagram4j.instagram4j.requests.IGPostRequest;
+import com.github.instagram4j.instagram4j.responses.IGResponse;
+import com.github.instagram4j.instagram4j.responses.accounts.AccountsUserResponse;
+
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Accessors;
+
+@RequiredArgsConstructor
+public class AccountsEditStatusRequest extends IGPostRequest<IGResponse> {
```

Comment:
Likewise, I do see constructing a payload and passing into constructor is perfectly fine.

I would like to see more constructors that let us set fields directly and have default fields.
It would not have to be as verbose when creating it.
Some thing like ```new AccountsEditStatusRequest(String text, String emoji, long expires_at)``` which have default field values (false notify and manual status type)
and the same for all fields.